gym-unity/setup.py

```diff
@@ -23,7 +23,7 @@ def run(self):
         tag = os.getenv("CIRCLE_TAG")
 
         if tag != EXPECTED_TAG:
-            info = "Git tag: {0} does not match the expected tag of this app: {1}".format(
+            info = "Git tag: {} does not match the expected tag of this app: {}".format(
```

Comment:
should we convert this to an actual f-string? `f"Git tag:...`

Reply:
I was saving that for the next PR. Adding `--py36-plus` should convert these automatically. The only ones I rewrote were the ones in environment.py (because one of them was broken).